examples/src/memory/azure_cosmosdb_nosql.ts

```diff
@@ -0,0 +1,47 @@
+import { ChatOpenAI } from "@langchain/openai";
+import { ConversationChain } from "langchain/chains";
+import { BufferMemory } from "langchain/memory";
+import {
+  AzureCosmsosDBNoSQLChatMessageHistory,
+  AzureCosmosDBNoSQLChatMessageHistoryInput,
+} from "@langchain/azure-cosmosdb";
+
+const input: AzureCosmosDBNoSQLChatMessageHistoryInput = {
+  sessionId: "<unique-session-id>",
+  userId: "user-id",
+  databaseName: "DATABASE_NAME",
+  containerName: "CONTAINER_NAME",
+};
+const memory = new BufferMemory({
+  chatHistory: new AzureCosmsosDBNoSQLChatMessageHistory(input),
+});
+
+const model = new ChatOpenAI({
+  model: "gpt-3.5-turbo",
+  temperature: 0,
+});
+
+const chain = new ConversationChain({ llm: model, memory });
```

Comment:
We generally don't suggest using this - prefer `RunnableWithMessageHistory` instead:

https://js.langchain.com/docs/versions/migrating_memory/#runnablewithmessagehistory-and-basechatmessagehistory

Note that we are suggesting LangGraph.js for apps that require memory in general